ConcernsCaseWork/ConcernsCaseWork.Data/Migrations/20230622093920_IncreaseSrmaNotes5000.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace ConcernsCaseWork.Data.Migrations
+{
+    /// <inheritdoc />
+    public partial class IncreaseSrmaNotes5000 : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<string>(
```

Comment:
It makes the column varchar max for anything that is greater than 4000 characters